scripts/update-drafts.ts

```diff
@@ -1,13 +1,29 @@
 import { Compat } from "compute-baseline/browser-compat-data";
 import fs from "node:fs/promises";
+import fsSync from "node:fs";
+import { fdir } from "fdir";
+import Path from "path";
 import { fileURLToPath } from "node:url";
 import webSpecs from "web-specs" assert { type: "json" };
 import { Document } from "yaml";
+import yargs from "yargs";
 
 import { features } from "../index.js";
 
 type WebSpecsSpec = (typeof webSpecs)[number];
 
+const argv = yargs(process.argv.slice(2))
+  .scriptName("update-drafts")
+  .usage("$0", "Update draft features with BCD keys mentioned in specs.")
+  .option("keys", {
+    type: "array",
+    describe: "Keys to match",
+  })
+  .option("paths", {
+    type: "array",
+    describe: "Draft feature files to update",
+  }).argv;
+
```

Comment:
So CLIs are always more complicated than I imagine. 😅 

This behaves very strangely if both options are set. I think it's applying both filters, such that nothing gets to match? I'm not sure.

I think the right thing to do here is to make sure the options don't layer on top of each other (or set some kind of flag like `--matches=id`, `--matches=path`, and then all positional arguments must be one type). But this is starting to get complicated and holding up useful work.

How about this: would you like to revert the changes after my review, add a `# TODO:` linking back here and one of us can take that up in a follow up PR?

Reply:
It was a simple change to make it an AND- for instance, `npx tsx scripts/update-drafts.ts --keys css --paths features/draft/spec/ba*` will update all CSS specs AND `badging.yml` and `battery-status.yml`.

If this doesn't seem correct, I'll revert and TODO.

Reply:
Nearly there! See my comment below.